content/javascript/concepts/promise/terms/all/all.md

```diff
@@ -1,6 +1,10 @@
 ---
 Title: '.all()'
-Description: "Returns a new Promise that can be accessed as an array of resolved values of fulfulled Promises. It takes an iterable object, such as an Array, that contains one or more Promise objects. This is ideal when working with Promises that depend on one another's completion."
+Description: "The .all() method in JavaScript Promises helps us work with multiple tasks at once. It takes a group of tasks (Promises) and returns a new Promise.
+
+When we use .all(), all the tasks start together, and .all() waits for all of them to finish. It gives us a Promise that contains the results of all the tasks.
```

Comment:
[Proofreading/Nitpick] According to the [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#writing-entries), pronouns(e.g., I, we, us, our, they, you, he, she) can't be used. 
```suggestion
When using `.all()`, the tasks start together and wait for all of them to finish. `.all()` gives a Promise that contains the results of all the tasks.
```